api/RTCRtpTransceiver.json

```
@@ -67,6 +67,40 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "value_stopped": {
+          "__compat": {
+            "description": "<code>\"stopped\"</code> value supported",
```

We usually avoid wording like "support" etc. in BCD descriptions. BCD is all about support, so no need to make the description longer than needed.

```suggestion
            "description": "<code>\"stopped\"</code> value",
```